### PR TITLE
Project 3 - Petra Almgren

### PR DESCRIPTION
**Pull request by petrame**
I did some of the stretch goals like icons in accordion, validation to the form, hovereffects, animations and a header on top of the video.

It took time to come up with a good business idea and a design, it's not something I have worked with before so i did a lot of research and thinking to come up with something I was satisfied with.
It was a bit tricky to get sticky navbar and video hero to work together.

### CODE REVIEW

**Review by bealun — CHANGES_REQUESTED** [review]
Hey! To be able to review your project properly I need a link to your deployed project, the live site on Netlify. Right now it's only linking to your GitHub. Please update your README and I'll then have a look at your code 👋

**Comment by petrame** [comment]
Ok, sorry about that. Now I have updated with the correct address.


Från: Beatrice Lundevall <notifications@github.com>
Skickat: den 13 september 2020 17:29
Till: Technigo/project-business-site <project-business-site@noreply.github.com>
Kopia: petrame <petra.eriksson@live.com>; Author <author@noreply.github.com>
Ämne: Re: [Technigo/project-business-site] Project 3 - Petra Almgren (#102)


@bealun requested changes on this pull request.

Hey! To be able to review your project properly I need a link to your deployed project, the live site on Netlify. Right now it's only linking to your GitHub. Please update your README and I'll then have a look at your code 👋

File: code/index.html [hunk 1]
```diff
@@ -3,28 +3,190 @@
...
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Oswald:wght@700&family=PT+Sans&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/e056cb308d.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="shortcut icon" href="./img/favicon.ico" type="image/x-icon" />
```
Comment (bealun) [inline code/index.html:16]: YEYY for a favicon and image folder - file structure is important! 👌

File: code/index.html [hunk 2]
```diff
@@ -3,28 +3,190 @@
...
+    <link rel="icon" href="./img/favicon.ico" type="image/x-icon" />
   </head>
+
   <body>
+    <!-- <header> -->
+    <!-- Sticky navbar -->
+    <nav class="navbar">
+      <a href="#"
+        ><img src="./img/logo.png" alt="" width="80px" height="80px"
```
Comment (bealun) [inline code/index.html:25]: Remember to use the 'alt' attribute, it fills a great purpose and especially when it comes to accessibility. Here's some reading:
https://www.w3schools.com/tags/att_img_alt.asp

File: code/index.html [hunk 3]
```diff
@@ -3,28 +3,190 @@
...
+    <section class="hero">
+      <video loop muted autoplay preload="auto" style="pointer-events: none">
+        <source src="./videos/video.mp4" type="video/mp4" />
+        Your browser does not support the video tag.
+      </video>
+      <div class="hero-text-container">
+        <h1>Cordonnerie</h1>
+        <h2>we take care of your shoes</h2>
+      </div>
+    </section>
```
Comment (bealun) [inline code/index.html:53]: Well-structured code 🙌

File: code/index.html [hunk 4]
```diff
@@ -3,28 +3,190 @@
...
+      <!-- Signup form -->
+      <form
+        class="main-section form"
+        action="http://httpbin.org/anything"
+        method="POST"
+      >
+        <p>Create your account to get started:</p>
+        <div class="form-item">
+          <!-- <label for="email">Email</label> -->
+          <input id="email" type="email" name="email" placeholder="Email" />
```
Comment (bealun) [inline code/index.html:72]: Is there a reason why you're not using the label? If not remember to remove unused code!

File: code/index.html [hunk 5]
```diff
@@ -3,28 +3,190 @@
...
+        </div>
+        <div class="form-item">
+          <!-- <label for="name">Password</label> -->
+          <input
+            id="password"
+            type="password"
+            name="password"
+            placeholder="Password"
+            minlength="8"
+          />
```
Comment (bealun) [inline code/index.html:82]: Nice to see some simple validation 🌟 If you don't want to submit empty input, you can check this out:
https://www.w3schools.com/tags/att_input_required.asp

File: code/index.html [hunk 6]
```diff
@@ -3,28 +3,190 @@
...
+            How can Cordonnerie help me?
+          </h3>
+          <div class="accordion-text-container">
+            <p class="accordion-text">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec
+              ullamcorper aliquet est, a tincidunt elit facilisis in. Proin
+              scelerisque facilisis eros id ultricies.
+            </p>
+          </div>
+        </div>
```
Comment (bealun) [inline code/index.html:120]: This is really good structure - keep it up!

File: code/style.css [hunk 7]
```diff
@@ -1,7 +1,291 @@
...
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: bold;
+}
+
+h1,
+h2,
+h3 {
+  font-family: "Oswald", sans-serif;
+}
```
Comment (bealun) [inline code/style.css:30]: Repeating myself but again, this is so well-structured!!

File: code/style.css [hunk 8]
```diff
@@ -1,7 +1,291 @@
...
+a,
+.accordion-text-container p {
+  font-size: 12px;
+}
+
+/* STICKY NAVBAR */
+
+.navbar {
+  z-index: 15;
+  position: sticky;
```
Comment (bealun) [inline code/style.css:45]: NICE, this is a cool styling feature 💥

File: code/style.css [hunk 9]
```diff
@@ -1,7 +1,291 @@
...
+.navbar a {
+  margin-left: 20px;
+  transition: background 0.6s ease;
+}
+
+.navbar li a:hover {
+  cursor: pointer;
+  background: #8d4501;
+  border-radius: 3px;
+}
```
Comment (bealun) [inline code/style.css:77]: We do love hover effects 👏

File: code/style.css [hunk 10]
```diff
@@ -1,7 +1,291 @@
...
+
+.accordion-item {
+  border: 1px solid #000;
+  background: #fff;
+  color: #000;
+  border: 2px solid #999;
+  border-radius: 8px;
+  margin: 16px 0;
+  box-shadow: 5px 5px 8px 0px rgba(0, 0, 0, 0.4);
```
Comment (bealun) [inline code/style.css:193]: Great job playing around with styling, if you wanna dig deeper here's something to look at:
https://tobiasahlin.com/blog/layered-smooth-box-shadows/

File: code/style.css [hunk 11]
```diff
@@ -1,7 +1,291 @@
...
+  transition: transform 0.3s ease-in-out;
+}
+.accordion-header.active::after {
+  transform: rotate(45deg);
+}
+
+footer {
+  background: #3f1f00;
+  display: flex;
+}
```
Comment (bealun) [inline code/style.css:234]: LOVE this! Perfect use of 'content', smoooooth transition and that nice transformation 🥳

File: code/style.css [hunk 12]
```diff
@@ -1,7 +1,291 @@
...
+
+  .accordion {
+    width: 40%;
+    flex-grow: 1;
+  }
+  
+  a,
+.accordion-text-container p {
+  font-size: 16px;
+}
```
Comment (bealun) [inline code/style.css:290]: It's very well done when you only need this small amount of code to create a fully responsive website 🌟

**Review by bealun — APPROVED** [review]
What an amazing business site you've built, WOW! 🤩 It's so nice to see that you're not afraid to explore and experiment with styling features, stay curious! You've got some great code structure and a super sleek layout. I do wanna encourage you to push your code more often, like 3 to 4 times per day, and with informative commit messages about what you're working on. This is standard practise in tech and will only benefit you in the long run. 
I hope you feel super proud of yourself and the work you've put in this week! 🎉

**Pull request closed without merging**